PopPUNK/refine.py

```diff
@@ -345,14 +416,21 @@ def newNetwork(s, sample_names, distMat, start_point, mean1, gradient,
 
     # Make network
     boundary_assignments = poppunk_refine.assignThreshold(distMat, slope, x_max, y_max, cpus)
-    G = constructNetwork(sample_names, sample_names, boundary_assignments, -1, summarise = False,
+    G = constructNetwork(sample_names, sample_names, boundary_assignments, -1,
+                            summarise = False,
+                            G_df = potential_edges_df,
```

Comment:
I don't get why this would be passed here, and think this needs to be rewritten to remove this

Reply:
See explanation above.